stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamDimens.kt

```diff
@@ -222,6 +232,8 @@ constructor(
         groupAvatarInitialsXOffset: Dp,
         groupAvatarInitialsYOffset: Dp,
         attachmentsContentImageMaxHeight: Dp,
+        attachmentsContentGiphyMaxWidth: Dp = attachmentsContentGiphyWidth,
```

Comment:
This is done in order to avoid a breaking change.

The new default mode is adaptive resizing so we could give users a better experience, and while `attachmentsContentGiphyWidth` and `attachmentsContentGiphyHeight` are ignored using this sizing mode, the maximum values are not.

Having the max values automatically use the regular values as default values means that even when we switch, whatever the customers previously passed in for width and height will still be applied as a constraint.